policyengine_us/reforms/ctc/ctc_older_child_supplement.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_us.model_api import *
+
+
+def create_ctc_older_child_supplement() -> Reform:
+    class ctc_child_individual_maximum(Variable):
+        value_type = float
+        entity = Person
+        label = "CTC maximum amount (child)"
+        unit = USD
+        documentation = (
+            "The CTC entitlement in respect of this person as a child."
+        )
+        definition_period = YEAR
+        reference = (
+            "https://www.law.cornell.edu/uscode/text/26/24#a",
+            "https://www.law.cornell.edu/uscode/text/26/24#h",
+            "https://www.law.cornell.edu/uscode/text/26/24#i",
+        )
+
```

Comment:
Can replace below with this right?
```suggestion
        defined_for = "is_tax_unit_dependent"

```